vertx-web/src/test/java/io/vertx/ext/web/WebTestBase.java

```diff
@@ -125,7 +125,7 @@ protected void testRequestBuffer(HttpClient client, HttpMethod method, int port,
         latch.countDown();
       } else {
         resp.bodyHandler(buff -> {
-          assertEquals(responseBodyBuffer, buff);
+          assertEquals(responseBodyBuffer, normalizeLineEndingsFor(buff));
```

Comment:
Sorry for being annoying :) what about having a flag to enable this on demand? For example if one is testing file downloads with binary payload this could again modify the payload.